### PR TITLE
Fvk add dcp developments

### PR DESCRIPTION
**Pull request by fvankrieken**

### CODE REVIEW

File: library/templates/dcp_developments.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+dataset:
+  name: &name dcp_developments
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: https://nyc3.digitaloceanspaces.com/edm-publishing/db-developments/main/{{ version }}/output/devdb.csv
```
Comment (mbh329) [inline library/templates/dcp_developments.yml:7]: What was the behavior of devdb when you ingested the shapefile? Did it just not work?

Comment (fvankrieken) [reply library/templates/dcp_developments.yml:7]: It did work, but output had slightly different column names than expected (lack of underscore in one column at least)

File: library/templates/dcp_developments.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      - dcp_councildistricts
+      - dcp_firecompanies
+      - doe_school_subdistricts
+      - doe_eszones
+      - doe_mszones
+      - dcp_policeprecincts
+      - hpd_hny_units_by_building
+      - dob_jobapplications
+      - dob_permitissuance
+      - dob_cofos
```
Comment (fvankrieken) [inline library/templates/dcp_developments.yml:56]: Question @mbh329 - seems like this is listing the datasets going into `db-developments`, not the downstream dependents on it, no? The latter being the intent of this part of the template?

Comment (mbh329) [reply library/templates/dcp_developments.yml:56]: I think each engineer thought of this section a little differently when we being more cognizant of adding "dependents" to the yaml files, perhaps it makes sense to have both though moving forward and renaming this section? That way its a bit easier to track which are the upstream vs downstream dependents

**Review by mbh329 — APPROVED** [review]

**Pull request merged**